app/plugin_api_test.go

```diff
@@ -1680,24 +1681,54 @@ func TestPluginHTTPUpgradeWebSocket(t *testing.T) {
 	}
 }
 
+type MockSlashCommandProvider struct {
```

Comment:
I guess it is from a "bad merge" o something like that, I'll investigate it, but probably is safe to remove it.

Reply:
Now I remember this, I added this because the "app" doesn't have by itself the `slashcommands` package imported, so it is not registering the slashcommands, so this was failing (but was not failing because we were checking the wrong type), so now, the type is right and it shows up. A slash command is needed to test it, so I created one and registered it.

Reply:
Ah got it. Thanks.